src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -3957,7 +3957,7 @@ public override int Emit(ProtoCore.AST.Node codeBlockNode, ProtoCore.Associative
             AllocateContextGlobals();
 
             core.startPC = this.pc;
-            if (core.ExecMode == ProtoCore.DSASM.InterpreterMode.kExpressionInterpreter)
+            if (core.Options.RunMode == ProtoCore.DSASM.InterpreterMode.kExpressionInterpreter)
```

Comment:
A question, why compiler core needs this ``RunMode``? Seems, well, ``RunMode`` is runtime stuff.